build/lint.cloudbuild.yaml

```diff
@@ -14,6 +14,9 @@
 
 timeout: 600s
 steps:
+- id: 3-networks-prep
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'chmod 777 app-foundation/3-networks-extension/merge_3-networks.sh && ./app-foundation/3-networks-extension/merge_3-networks.sh && echo "Merge Success"']
```

Comment:
You can set executable permissions `chmod +x app-foundation/3-networks-extension/merge_3-networks.sh` and push the updated file to git which will remove the need to do it in pipeline